Wordle.Api/Wordle.Api/Identity/Roles.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Wordle.Api.Identity
+{
+    public static class Roles
+    {
+        public const string Admin = "Admin";
```

Comment:
possibly need to add the roles for master and over 21, depends on how you implement